src/components/icon-extention-with-title/IconExtentionWithTitle.tsx

```diff
@@ -1,29 +1,28 @@
 import { FC } from 'react'
-import { useTranslation } from 'react-i18next'
 
 import Box from '@mui/material/Box'
+
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import { styles } from '~/components/icon-extention-with-title/IconExtentionWithTitle.styles'
 
+import { convertBytesToProperSize } from '~/utils/bytes-to-proper-size'
+
 interface IconExtentionWithTitleProps {
   title: string
-  description: number
+  description?: number
 }
 
 const IconExtentionWithTitle: FC<IconExtentionWithTitleProps> = ({
   title,
   description
 }) => {
-  const { t } = useTranslation()
   const fileExtension = title.slice(title.lastIndexOf('.') + 1)
 
   return (
     <Box sx={styles.container}>
       <Box sx={styles.iconBox}>{fileExtension}</Box>
       <TitleWithDescription
-        description={
-          description > 0 && `${description} ${t('common.megabytes')}`
-        }
+        description={description && convertBytesToProperSize(description)}
```

Comment:
I suspect the sonar thinks it's a bug here, that's why i did in previous pr (description > 0)